target/linux/generic/hack-6.12/932-fix-undefined-references.patch

```diff
@@ -0,0 +1,13 @@
+--- a/kernel/vmcore_info.c
++++ b/kernel/vmcore_info.c
```

Comment:
This patch should be squashed into `pending-6.12/203-kallsyms_uncompressed.patch`. As a part of `generic: 6.12: manually rebuilding pending patches`.

Reply:
Done